src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js

```diff
@@ -162,28 +198,50 @@ class ActivityFeed extends React.Component<Props, State> {
         this.setState({ mentionSelectorContacts: getMentionWithQuery(searchStr) });
     };
 
+    componentDidMount(): void {
+        const { comments, tasks, versions } = this.props;
+        this.sortFeedItems(comments, tasks, versions);
+    }
+
+    componentWillReceiveProps(nextProps: any): void {
+        const { comments, tasks, versions } = nextProps;
+
+        if (comments || tasks || versions) {
+            this.sortFeedItems(comments, tasks, versions);
+        }
+    }
+
+    sortFeedItems(comments?: Comments, tasks?: Tasks, versions?: FileVersions): void {
+        const items = [];
```

Comment:
If you default these params to [] you could consolidate this to 

items = [...comments, ...tasks, ...versions, ...pendingItems];

Reply:
done